features/identity-core/org.wso2.carbon.identity.core.server.feature/resources/org.wso2.carbon.identity.core.server.feature.default.json

```diff
@@ -72,6 +72,8 @@
   "oauth.token.validation.validation_response_jwt_validity": "15m",
   "oauth.token_generation.retry_count_on_persistence_failures": "5",
 
+  "oauth.access_token.generate_with_sp_tenant_domain": true,
```

Comment:
Please update the identity.xml value as well

Reply:
This is updated.